public/browse.css

```
@@ -32,6 +32,30 @@ body {
   grid-template-rows: auto auto auto auto 1fr auto;
 }
 
+/*
+  Website Header
+*/
+
+m-website-header .website-header-inner-container {
+  margin: 0 calc(-1 * var(--space-small));
+}
+
+@media only screen and (max-width: 720px) {
+  m-website-header .website-header-inner-container {
+    margin-top: var(--space-small);
+  }
+}
+
+m-website-header a {
+  color: white;
+  text-decoration: none;
+  padding: var(--space-small);
+}
```

```suggestion
m-website-header a {
  color: white;
  display: inline-block;
  text-decoration: none;
  padding: var(--space-xx-small) var(--space-small);
}
```
The `margin-top` on the container made the links not exactly centered. Adding `display: inline-block;` to the links makes up for the margin. The padding does add extra spacing on the bottom of the header, but that's not seen until around `474px`.